src/tools/rest-backend/config.hpp

```diff
@@ -0,0 +1,83 @@
+#ifndef ELEKTRA_REST_CONFIG_HEADER_GUARD
+#define ELEKTRA_REST_CONFIG_HEADER_GUARD
+
+// Enables the test moe for the rest service.
+#define ELEKTRA_REST_TESTMODE 1
+
+// Additional macros that are set for the test mode.
+// - Authentication is ignored
+#ifdef ELEKTRA_REST_TESTMODE
+#define ELEKTRA_REST_NO_AUTH
+#endif
+
+// Base path for the repository used by the rest service to store its data.
+// Path may NOT end with a slash.
+#define ELEKTRA_REST_CONFIG_REPOSITORY_PATH "dir/repository"
```

Comment:
The idea was that some of this values could be changed through CMake, especially paths.